docs/topics/cli-run.adoc

```diff
@@ -110,22 +110,22 @@ The following commands allow Bash completion to persist across restarts:
 +
 [source,terminal]
 ----
-source <MTA_HOME>/bash-completion/mta-cli
+source <MTR_HOME>/bash-completion/windup-cli
```

Comment:
Missing prompt ($)

Reply:
@CarmiWisemon -- Some of the older docs don't have the $ prompt in the commands (or the root prompt, if needed) but you should always add one if it is missing.

Reply:
@apinnick: Any problem with creating an attribute, say {lc_spn}, that represents the product acronym written in lower case?

So that  `source <MTA_HOME>/bash-completion/mta-cli` would then be written `source <{ShortProductName}_HOME>/bash-completion/{lc_spn}-cli`   _if_  we were not making the change that Philip suggested to `...windup-cli`

Reply:
Corrected.